core/src/main/java/nl/nn/adapterframework/doc/model/ConfigChild.java

```diff
@@ -113,6 +113,29 @@ public int compareTo(ElementChild other) {
 		return CONFIG_CHILD_COMPARATOR.compare(this, (ConfigChild) other);
 	}
 
+	/**
+	 * Registers the syntax 1 name of this {@link ConfigChild} with the
+	 * {@link ElementType}. This is done recursively because the XSD does
+	 * not only use the declared config children, but also the inherited
+	 * config children of a {@link FrankElement}. For each combination of
+	 * a syntax 1 name and an {@link ElementChild}, we have &lt;xs:group&gt;
+	 * declarations in the XSD. The recursion ensures that references to
+	 * ancestor groups are valid.
+	 */
```

Comment:
Volgens mij is recursie niet nodig. Uit het ElementType genereer je groepen voor elke syntax1naam. Die groepen bevatten een choice van elementen met een samengestelde naam (bijv. JavaErrorSender) en een verwijzing naar een complex-type. Die complex-types zijn voor elke syntax1naam binnen het ElementType hetzelfde, en kunnen dus gedeeld worden.

Reply:
Je hebt gelijk. Ik heb het aangepast.